packages/docs/.vuepress/plugins/plugin-choices/clientRootMixin.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  updated() {
+    import('choices.js').then((module) => {
```

Comment:
I had this previously, but it would cause build failures. This approach comes from the VuePress Docs https://v1.vuepress.vuejs.org/guide/using-vue.html#browser-api-access-restrictions